prac_07/guitars.py

```diff
@@ -1,7 +1,7 @@
 """
 CP1404 Practical
 Uses a list to store all guitars from .csv file and also stores the user's guitars and
-keeps inputting until they enter a blank name then writes back to .csv file.
+keeps inputting until they enter a blank name then writes back to .csv file..
 """
 
```

Comment:
```
    in_file = open('guitars.csv', 'r')  # File format is like: Name,Year,Cost
    for line in in_file:
        parts = line.strip().split(',')
        guitar = Guitar(parts[0], int(parts[1]), float(parts[2]))
        guitars.append(guitar)
    in_file.close()
    ```
    
 This block of code looks complex enough that I believe it could be its own function, as it would take some of the high-level detail out of the main function. ⚙️